yt_dlp/extractor/svt.py

```diff
@@ -429,24 +469,27 @@ def suitable(cls, url):
     def _real_extract(self, url):
         path, display_id = self._match_valid_url(url).groups()
 
```

Comment:
we only need to match the id now
```suggestion
    def _real_extract(self, url):
        display_id = self._match_id(url)
```